<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ScriptsTool</class>
 <widget class="QDialog" name="ScriptsTool">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>492</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Scripts</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="tabBarAutoHide">
      <bool>true</bool>
     </property>
     <widget class="QWidget" name="scriptsTab">
      <attribute name="title">
       <string>Scripts</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_3">
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>LoadedScripts</string>
           </property>
           <property name="buddy">
            <cstring>scripts</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QListWidget" name="scripts">
           <property name="contextMenuPolicy">
            <enum>Qt::CustomContextMenu</enum>
           </property>
           <property name="spacing">
            <number>1</number>
           </property>
           <property name="sortingEnabled">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QPushButton" name="addScripts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>AddScripts</string>
             </property>
             <property name="accessibleName">
              <string>AddScripts</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="class" stdset="0">
              <string notr="true">icon-plus</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="removeScripts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>RemoveScripts</string>
             </property>
             <property name="accessibleName">
              <string>RemoveScripts</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="class" stdset="0">
              <string notr="true">icon-trash</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="reloadScripts">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="toolTip">
              <string>ReloadScripts</string>
             </property>
             <property name="accessibleName">
              <string>ReloadScripts</string>
             </property>
             <property name="text">
              <string notr="true"/>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
             <property name="class" stdset="0">
              <string notr="true">icon-refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="defaults">
             <property name="text">
              <string>Defaults</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="editScript">
             <property name="text">
              <string>EditScript</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="scriptLog">
             <property name="text">
              <string>ScriptLogWindow</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QVBoxLayout" name="propertiesLayout">
         <item>
          <widget class="QLabel" name="label_3">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Maximum">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="text">
            <string>Description</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="description">
           <property name="text">
            <string notr="true"/>
           </property>
           <property name="wordWrap">
            <bool>true</bool>
           </property>
           <property name="margin">
            <number>12</number>
           </property>
           <property name="openExternalLinks">
            <bool>false</bool>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="pythonSettingsTab">
      <attribute name="title">
       <string>PythonSettings</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="pythonPathLabel">
         <property name="text">
          <string notr="true"/>
         </property>
         <property name="buddy">
          <cstring>pythonPath</cstring>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLineEdit" name="pythonPath">
           <property name="readOnly">
            <bool>true</bool>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pythonPathBrowse">
           <property name="accessibleName">
            <string>PythonSettings.BrowsePythonPath</string>
           </property>
           <property name="text">
            <string>Browse</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QLabel" name="pythonVersionLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>510</width>
           <height>306</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="close">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>close</tabstop>
  <tabstop>pythonPath</tabstop>
  <tabstop>pythonPathBrowse</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
